From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Artem Butusov <art.sormy@gmail.com>
Date: Mon, 12 Oct 2026 02:00:00 -0400
Subject: [PATCH] media: meson: vdec: build v4l2-h264 in

The G12A H.264 codec uses the v4l2-h264 reference-list helpers, which
Armbian's meson64 kernel does not build. The fetch script stages
v4l2-h264.c from the matching kernel source.

Signed-off-by: Artem Butusov <art.sormy@gmail.com>
---
 vdec/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --color -ruN C/vdec/Makefile D/vdec/Makefile
--- a/vdec/Makefile
+++ b/vdec/Makefile
@@ -4,7 +4,7 @@
 meson-vdec-objs = esparser.o vdec.o vdec_helpers.o vdec_platform.o
 meson-vdec-objs += vdec_1.o vdec_hevc.o
 meson-vdec-objs += codec_mpeg12.o codec_mpeg12_multi.o codec_h264.o codec_h264_g12a.o
-meson-vdec-objs += codec_h264_g12a_dpb.o
+meson-vdec-objs += codec_h264_g12a_dpb.o v4l2-h264.o
 meson-vdec-objs += codec_hevc_common.o codec_vp9.o codec_hevc.o
 meson-vdec-$(CONFIG_VIDEO_MESON_VDEC_KUNIT_TEST) += codec_h264_g12a_dpb_test.o
 
-- 
2.47.0

